fixed mistake in split method doc comment (#141)
dart-lang/path: CHANGELOG.md

## 1.8.4-dev

* Require Dart 2.19

> * fix mistake in the split method doc comment

Reviewer: ```suggestion
* Fixed an issue with the `split` method doc comment.
```
